### PR TITLE
Add: maple-fonts

### PR DESCRIPTION
**Pull request by pugaizai**

### CODE REVIEW

File: anda/fonts/maple/maple-fonts.spec [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Version:        6.4
+Release:        %autorelease
+Summary:        Open source monospace & nerd font with round corner and ligatures.
+License:        OFL-1.1
+URL:            https://github.com/subframe7536/Maple-font
+Source0:        %{url}/releases/download/v%{version}/MapleMono-SC-NF.zip
+Source1:        %{url}/releases/download/v%{version}/MapleMono-ttf.zip
+Source2:        %{url}/releases/download/v%{version}/MapleMono-NF.zip
+Source3:        https://raw.githubusercontent.com/subframe7536/maple-font/main/OFL.txt
+BuildRequires:  fontpackages-devel
```
Comment (madonuko) [inline anda/fonts/maple/maple-fonts.spec]: I'm a bit confused, what is this for?

**Comment by madonuko** [comment]
oh yeah, `%define debug_package %nil` at the top by the way

File: anda/fonts/maple/maple-fonts.spec (hunk 1 above)
Comment (pugaizai) [reply anda/fonts/maple/maple-fonts.spec]: > I'm a bit confused, what is this for?

actually it is not needed.

File: anda/fonts/maple/maple-fonts.spec [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Version:        6.4
+Release:        %autorelease
+Summary:        Open source monospace & nerd font with round corner and ligatures.
+License:        OFL-1.1
+URL:            https://github.com/subframe7536/Maple-font
+Source0:        %{url}/releases/download/v%{version}/MapleMono-SC-NF.zip
+Source1:        %{url}/releases/download/v%{version}/MapleMono-ttf.zip
+Source2:        %{url}/releases/download/v%{version}/MapleMono-NF.zip
+Source3:        https://raw.githubusercontent.com/subframe7536/maple-font/main/OFL.txt
+
```
Comment (madonuko) [inline anda/fonts/maple/maple-fonts.spec]:
```suggestion
BuildArch:      noarch
```

**Review by madonuko — APPROVED** [review]

**Comment by madonuko** [comment]
Oops, sign commits moment.

**Pull request closed without merging**